nncf/experimental/common/tensor_statistics/statistics.py

```diff
@@ -24,6 +24,22 @@ class TensorStatistic:
 
     TENSOR_STATISTIC_OUTPUT_KEY = "tensor_statistic_output"
 
+    def get_data(self):
+        return {key: getattr(self, key) for key in self.keys()}
+
+    def load_data(self, data: Dict):
+        for key in self.keys():
+            setattr(self, key, data.get(key))
+
+    @classmethod
+    def from_kwargs(cls, kwargs: Dict[str, Any]) -> TensorStatistic:
+        args = {key: kwargs[key] for key in cls.keys()}  # noqa: SIM118
+        return cls(**args)
+
+    @classmethod
+    def keys(cls):
+        return []
```

Comment:
Ultra minor: can we use tuple here instead of the list to specify that keys could not be modified anywhere?

Reply:
done